### PR TITLE
Modify render function

### PR DESCRIPTION
**Pull request by vibhoooo**
Ticket ->

LB-1127
https://tickets.metabrainz.org/browse/LB-1211

I modified the render function to get the details of the listen.

### CODE REVIEW

File: listenbrainz/config.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
+import os
```
Comment (amCap1712) [inline listenbrainz/config.py:1]: This file should not be committed.

**Review by amCap1712 — COMMENTED** [review]
Hi! Thanks for starting work on this. However, this still needs some work. Did you try running the website locally and seeing the UI changes work as expected? If not, try to do that if you get stuck feel free to me or monkey for help in IRC.

**Comment by MonkeyDo** [comment]
Thank you for opening this pull request @vibhoooo !

I haven't treid the code yet, but I wanted to give you a heads up: you committed the file listenbrainz/config.py, which we never want to commit. This file is for your use only and wouldn't make sense to add it to the repository.

You can [cherry-pick](https://git-scm.com/docs/git-cherry-pick) the first commit and open a new PR or force-push to this one.

As a matter of fact, there are some secret tokens in there that are now public, so you're gonna want to delete and recreate the associated applications: [musicbrainz app](https://github.com/metabrainz/listenbrainz-server/pull/2334/files#diff-ebdf0156fea5cdf0d18d03cbcc5763d520188b6af947302cc63a00800bd95213R84), [LastFM app](https://github.com/metabrainz/listenbrainz-server/pull/2334/files#diff-ebdf0156fea5cdf0d18d03cbcc5763d520188b6af947302cc63a00800bd95213R88), [Spotify app](https://github.com/metabrainz/listenbrainz-server/pull/2334/files#diff-ebdf0156fea5cdf0d18d03cbcc5763d520188b6af947302cc63a00800bd95213R156) and [CritiqueBrainz app](https://github.com/metabrainz/listenbrainz-server/pull/2334/files#diff-ebdf0156fea5cdf0d18d03cbcc5763d520188b6af947302cc63a00800bd95213R161).

Make sure you delete the applicatio,ns so that nobody can do things in your name now that the ids and tokens are public.

File: listenbrainz/config.py (hunk 1 above)
Comment (vibhoooo) [reply listenbrainz/config.py:1]: Okay.
I will remove it.

File: frontend/js/src/mbid-mapping/MbidMappingModal.tsx [hunk 2]
```diff
@@ -136,6 +139,10 @@ export default class MbidMappingModal extends React.Component<
                 recording, you can do so manually here. Paste a Recording MBID
                 to link it to this Listen and all others with the same metadata.
               </p>
+              
+              <div title={trackName} className="ellipsis-2-lines"></div>
```
[the remark this answers is not in the document]
Comment (vibhoooo) [reply frontend/js/src/mbid-mapping/MbidMappingModal.tsx:143]: Sorry.
I am a bit new to it.
Actually I tried to take some help from the Listen.tsx file.
I will make the required changes and update it.

**Comment by vibhoooo** [comment]
@MonkeyDo 
Sorry.
I was not aware of that.
I tried to put it as tests were failing but it got included in the PR.
Yes I will remove it.
Sorry again.

**Comment by vibhoooo** [comment]
@amCap1712 
I was learning but I tried to implement it in between...my bad.
I will try to work on my mistakes 
I tried running the frontend tests 1 test was failing but I did not see the implementation in the local website.
I will keep this in mind from now onwards.
I will try to make all the required changes and update things accordingly.
Thanks for review.

**Pull request closed without merging**